Does not install on Linux

Followed instructions, ran npm run build-client.

Running npm install:
```
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
```

Running npm run build-client:
```arc@windowssucks:~/.config/BetterDiscordApp-2$ npm run build-client

> betterdiscord@0.3.0 build-client /home/arc/.config/BetterDiscordApp-2
> webpack --progress --colors

 10% building modules 1/1 modules 0 active(node:30720) DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
Hash: a807959fa170203e64ce                                                            
Version: webpack 2.5.1
Time: 4984ms
                  Asset    Size  Chunks                    Chunk Names
betterdiscord.client.js  593 kB       0  [emitted]  [big]  main
   [0] ./src/client/vendor/index.js 630 bytes {0} [built]
   [3] ./src/client/modules/logger.js 4.65 kB {0} [built]
   [4] ./src/client/modules/utils.js 3.71 kB {0} [built]
   [5] ./src/client/modules/index.js 984 bytes {0} [built]
   [6] ./src/client/modules/events.js 1.9 kB {0} [built]
  [10] ./src/client/modules/cache.js 1.68 kB {0} [built]
  [11] ./src/client/modules/ipc.js 2.57 kB {0} [built]
  [20] ./src/client/modules/dom.js 2.47 kB {0} [built]
  [21] ./src/client/modules/reflection_deep.js 5.52 kB {0} [built]
  [22] ./src/client/modules/renderer.js 4.07 kB {0} [built]
  [36] ./src/client/vendor/jquery.js 457 bytes {0} [built]
  [38] ./src/client/vendor/reactdom.js 468 bytes {0} [built]
  [40] ./src/client/ui/index.js 506 bytes {0} [built]
  [41] ./src/scss/main.scss 284 kB {0} [built]
  [44] ./src/client/index.js 3.73 kB {0} [built]
    + 72 hidden modules

ERROR in ./src/client/modules/index.js
Module not found: Error: Can't resolve './browserEvents' in '/home/arc/.config/BetterDiscordApp-2/src/client/modules'
 @ ./src/client/modules/index.js 14:19-45
 @ ./src/client/index.js

ERROR in ./src/client/modules/index.js
Module not found: Error: Can't resolve './pluginManager' in '/home/arc/.config/BetterDiscordApp-2/src/client/modules'
 @ ./src/client/modules/index.js 23:19-45
 @ ./src/client/index.js

npm ERR! Linux 4.4.0-22-generic
npm ERR! argv "/home/arc/.nvm/versions/node/v6.10.3/bin/node" "/home/arc/.nvm/versions/node/v6.10.3/bin/npm" "run" "build-client"
npm ERR! node v6.10.3
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! betterdiscord@0.3.0 build-client: `webpack --progress --colors`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the betterdiscord@0.3.0 build-client script 'webpack --progress --colors'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the betterdiscord package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     webpack --progress --colors
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs betterdiscord
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls betterdiscord
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/arc/.config/BetterDiscordApp-2/npm-debug.log```

--- a/ISSUE_TEMPLATE.md
+++ b/ISSUE_TEMPLATE.md
@@ -1,0 +1,86 @@
+*Before* you create an issue, make sure you have:
+
+1. Checked that that issue hasn't already been submitted. You can search open issues [here](https://github.com/Jiiks/BetterDiscordApp/issues).
+2. Checked that there isn't an answer in either of the Discord servers (if you can access them - don't complain if you can't).
+3. Checked that it isn't in the following list of frequently asked questions.
+
+If none of that helps you, delete the prefilled contents of this issue and ask whatever question you may have *in English so we can understand it*.
+
+### FAQ
+
+> *BetterDiscord doesn't work.*  
+> *BetterDiscord won't install.*  
+> *BetterDiscord doesn't support macOS.*
+
+This branch isn't being updated anymore. Until BetterDiscord v2 is released you can use [Zach Rauen's fork](https://github.com/rauenzi/BetterDiscordApp).
+
+> *BetterDiscord doesn't support Linux.*
+
+There's an installer script for BetterDiscord on Linux available [here](https://github.com/bb010g/betterdiscordctl).
+
+> *BetterDiscord doesn't work in my browser.*
+
+The browser version doesn't exist anymore. Use the desktop client.
+
+> *BetterDiscord doesn't work on my phone.*
+
+... and never will. BetterDiscord is a client modification for Discord's desktop Electron client. (Electron is a framework for making cross platform apps using web technologies. Basically the desktop client is just the web client in a specialised web browser.)
+
+> *Is there any security risks involved with installing BetterDiscord?*  
+> *My friend was hacked after he/she installed BetterDiscord.*
+
+*Not by itself.* However, BetterDiscord allows loading plugins. Discord, BetterDiscord, and any plugins you install (even without enabling) have full access to your system as your local user account. (Including full access to your Discord account.) So, make sure you only download plugins from sources you trust.
+
+Also, please note that [BetterDocs](https://betterdocs.net) and other third-party sources by definition *are not* affiliated with BetterDiscord or it's developers.
+
+> *How do I install plugin [x]?*  
+> *Is there a plugin to [x]?*  
+> *Does [x] theme exist?*
+
+https://imgur.com/lczPQxW
+
+> *Minimal mode broke everything.*
+
+You can get back into Discord's settings to disable it by pressing `Control + ,` on Windows and Linux or `Command + ,` on macOS.
+
+> *How do I completely hide messages from people I've blocked?*
+
+Add this to your custom CSS:
+
+```css
+.message-group-blocked {
+    display: none !important;
+}
+```
+
+> *How do I remove BetterDiscord?*
+
+BetterDiscord is installed to the following locations:
+
+Platform     | Discord path
+-------------|---------------
+Windows      | `%APPDATA%`\discord\0.0.`*`\modules\discord_desktop_core
+macOS        | ~/Library/Application Support/discord/0.0.`*`/modules/discord_desktop_core
+Linux        | ~/.config/discord/0.0.`*`/modules/discord_desktop_core
+
+To remove BetterDiscord, open the file `index.js` in that directory and make sure only this line exists:
+
+```js
+module.exports = require('./core');
+```
+
+Then, delete the `core` directory.
+
+BetterDiscord's data is stored in the following locations:
+
+Platform     | BetterDiscord data path
+-------------|---------------
+Windows      | `%APPDATA%`\BetterDiscord
+macOS        | ~/Library/Preferences/BetterDiscord
+Linux        | ~/.config/BetterDiscord
+
+To remove all of BetterDiscord's data, just delete that directory.
+
+> *BetterDiscord v2?*
+
+[BetterDiscord v2 is the new version of BetterDiscord currently in development.](https://github.com/JsSucks/BetterDiscordApp) If you'd like to help out with BetterDiscord v2, join the JsSucks Discord server. (Invite code `KYKwv4R`.) The only requirement is JavaScript/ES6 knowledge.

